projects/cdk/tokens/is-cypress.ts

```diff
@@ -1,16 +1,22 @@
 import {inject, InjectionToken} from '@angular/core';
 import {WINDOW} from '@ng-web-apis/common';
 
+/**
+ * @deprecated use {@link TuiWindowWithCypress}
+ */
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export interface WindowWithCypress extends Window {
     Cypress?: unknown;
 }
 
+export interface TuiWindowWithCypress extends WindowWithCypress {}
+
 /**
  * {@link https://docs.cypress.io/faq/questions/using-cypress-faq#Is-there-any-way-to-detect-if-my-app-is-running-under-Cypress Cypress docs}
  */
 export const TUI_IS_CYPRESS = new InjectionToken<boolean>(
     'Detect if app is running under Cypress',
     {
-        factory: () => !!inject<WindowWithCypress>(WINDOW).Cypress,
+        factory: () => !!inject<TuiWindowWithCypress>(WINDOW).Cypress,
```

Comment:
I don't think this adds anything to the type-safety. We might just type it `any` as well. This way we just pollute exports.

Reply:
fixed